stock_orderpoint_generator/wizard/orderpoint_generator.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# © 2012-2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import _, api, fields, models
+
+from openerp.exceptions import UserError
+
+_template_register = ['orderpoint_template_id']
+
+
+class OrderpointGenerator(models.TransientModel):
+    """ Wizard defining stock.warehouse.orderpoint configurations for selected
+    products. Those configs are generated using templates
+    """
+
+    _name = 'stock.warehouse.orderpoint.generator'
+    _description = 'Orderpoint Generator'
+
+    orderpoint_template_id = fields.Many2many(
+        'stock.warehouse.orderpoint.template',
+        rel='order_point_generator_rel',
+        string='Stock rule template'
+    )
+
+    @api.multi
+    def action_configure(self):
+        """ Action to retrieve wizard data and launch creation of items.
+        """
+        self.ensure_one()
+
+        product_ids = self.env.context.get('active_ids')
+        assert product_ids and isinstance(product_ids, list)
+
+        if self.env.context.get('active_model') == 'product.template':
+            templates = self.env['product.template'].browse(product_ids)
+            product_ids = templates.mapped('product_variant_ids.id')
+            if len(product_ids) != len(templates):
+                raise UserError(_(
+                    'Cannot apply because some of selected '
+                    'products has multiple variants.'
```

Comment:
Not sure  to understand too :) !

But in V7, this wizard was only available in product.product tree view. In V9, most of products list are product.template so I added the same button on these views but I wasn't sure that users want selected templates to be applied on all variants.... 

Multiple variants may mean different orderpoints? I don't really know....